### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by iansk** [hlxsites/prisma-cloud-docs-website#38]
We translate content on a per-book basis. For example, consider the following doc set. We currently only translate the `admin guide`. Books like `release-notes` will never be translated. I think we need a mechanism to select language for specific books, not the entire site, which never be fully localized.

```
compute
├── admin-guide
├── is-notifications
├── ops-guide
├── public-sector
└── release-notes
```

**Comment by maxakuru** [comment]
since these books are already in the locale folders in docs, ie: https://github.com/hlxsites/prisma-cloud-docs/tree/main/docs/en

maybe it would be enough to redirect to `/en/...` for pages in there rather than removing the localization path segment entirely?

that would make paths more consistent and doesn't add a special case that needs to be handled/learned/understood

**Comment by iansk** [comment]
@maxakuru Yes, I think it would be enough. The repo layout is simple, and I'd like to keep it as-is. But what would the experience look on the site? Could we have a per-book language selector, where the selector only appears if there's localized content?

**Comment by maxakuru** [comment]
how about use the metadata sheet & a dropdown for a language picker, similar to the setup for book versions? we have some space in the UI below the version picker since the `download pdf` button was removed

it would mean adding another workbook too, like the `versions` one, to track the languages available for each book... maybe called.... `languages`?

**Comment by iansk** [comment]
Sounds good, let's try this approach.
